Plugins/Public/cloak_plugin/Cloak.cpp

```diff
@@ -962,6 +952,14 @@ int __cdecl Dock_Call(unsigned int const &iShip, unsigned int const &iDockTarget
 	return 0;
 }
 
+void __stdcall SystemSwitchOut(uint iClientID, FLPACKET_SYSTEM_SWITCH_OUT& switchOutPacket)
+{
+	// in case of SERVER_PACKET hooks, first argument is junk data before it gets processed by the server.

```

Comment:
what does this mean? iClientID Is wrong and that's why we're going to `HkGetClientIDByShip(switchOutPacket.shipId)` instead??